mobile/src/navigations/onboarding-navigator/talent-onboarding.tsx

```diff
@@ -28,8 +30,13 @@ import { Header, Steps } from './components/components';
 const Drawer = createDrawerNavigator<TalentOnboardingNavigationParameterList>();
 
 const TalentOnboardingNavigator: React.FC = () => {
-    const { completedStep } =
-        useAppSelector(({ talents }) => talents.onboardingData) ?? {};
+    const { dataStatus, onboardingData } = useAppSelector(
+        ({ talents }) => talents,
+    );
+
+    const { completedStep } = onboardingData ?? {};
+
+    const isPendingAuth = dataStatus === DataStatus.PENDING;
```

Comment:
Guess we should rename this variable. On something like isStepDataPending or something like that.